cypress/integration/executions_spec.js

```diff
@@ -0,0 +1,52 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Dashboard Executions Page', () => {
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit('/#/executions');
+      shouldBeRedirectedToLogin();
+    });
+  });
+
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.task('resetState');
+    });
+
+    after(() => {
+      cy.task('resetState');
+    });
+
+    it('should display a link to view executions', () => {
+      cy.visit('/');
+
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      cy.contains('.heading--large', 'Execution Overview');
+
+      // shows a summary count of completed and failed executions
+      cy.get('.overview-num__wrapper ul li')
+        .first().contains('li', '3 Completed')
+        .next().contains('li', '1 Failed')
+        .next().contains('li', '1 Running');
+
+      // shows a list of executions with IDs and status
+      cy.get('table tbody tr').as('list');
+      cy.get('@list').its('length').should('be.eq', 5);
+      cy.get('@list').each(($el, index, $list) => {
+        cy.wrap($el).children().as('children');
+        cy.get('@children').its('length').should('be.eq', 6);
+        cy.get('@children').eq(0).children('a').should('have.attr', 'href');
```

Comment:
Can we assert the value of the href attribute?

Reply:
I just realized that you're not doing this because the values will be different for each row. But can we just read the source JSON for the fake API fixture so we can compare against that? Right now these assertions are a little generic